helm/cluster-openstack/templates/cluster.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: cluster.x-k8s.io/v1alpha4
+kind: Cluster
+metadata:
+  annotations:
+    cluster.giantswarm.io/description: {{ .Values.description }}
+  labels:
+    release.giantswarm.io/version: {{ .Values.releaseVersion }}
+    giantswarm.io/cluster: {{ .Values.name }}
+    cluster.x-k8s.io/cluster-name: {{ .Values.name }}
+    giantswarm.io/organization: {{ .Values.organization }}
+  name: {{ .Values.name }}
+  namespace: org-{{ .Values.organization }}
```

Comment:
I'm not sure if this can be this simple. I'm thinking about organization name normalization: special chars, lowercase/uppercase, etc.

Reply:
Yes, I meant to change this. In some places we use `{{ .Release.Namespace }}`. I was wondering if we could not specify the namespace at all and let it be applied by `helm install`. I will need to experiment some more.

Reply:
Done. Discussed here https://gigantic.slack.com/archives/C02GDJJ68Q1/p1640009319149500